cmd/autobumper/main.go

```diff
@@ -0,0 +1,334 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// copied from go/src/k8s.io/test-infra/experiment/autobumper/main.go
+// originally based on: 3c673e642 (HEAD -> master, origin/master, origin/HEAD) Merge pull request #13978 from nilo19/t-qini-update_k8s_and_aks_engine_version_for_periodic_tests
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/test-infra/experiment/image-bumper/bumper"
+	"k8s.io/test-infra/prow/config/secret"
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/robots/pr-creator/updater"
+)
+
+const (
+	prowPrefix      = "gcr.io/k8s-prow/"
+	testImagePrefix = "gcr.io/k8s-testimages/"
+	prowRepo        = "https://github.com/kubernetes/test-infra"
+	testImageRepo   = prowRepo
+	oncallAddress   = "https://storage.googleapis.com/kubernetes-jenkins/oncall.json"
```

Comment:
well, we definitely don't need that.

Reply:
Just a starting point ... we will do our customization later.
But since you get confused too, I will start with an empty main.